paymentsheet/src/androidTest/java/com/stripe/android/paymentsheet/CustomerSheetTest.kt

```diff
@@ -255,6 +342,26 @@ internal class CustomerSheetTest {
         context.markTestSucceeded()
     }
 
+    private fun billingDetails(): RequestMatcher {
+        return RequestMatchers.composite(
+            bodyPart("billing_details%5Baddress%5D%5Bpostal_code%5D", CustomerSheetPage.ZIP_CODE),
+            bodyPart("billing_details%5Baddress%5D%5Bcountry%5D", CustomerSheetPage.COUNTRY)
+        )
+    }
+
+    private fun fullBillingDetails(): RequestMatcher {
+        return RequestMatchers.composite(
+            bodyPart("billing_details%5Bname%5D", urlEncode(CustomerSheetPage.NAME)),
```

Comment:
I like the urlEncode here. It makes it much easier to read. Should we urlEncode the key of the body part too? `urlEncode("billing_details[name]")`?